projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -941,12 +944,15 @@ export const getSidebarCardSlug = state => {
 		return 'upsell';
 	}
 
-	if ( 'awaiting_credentials' === rewindState && ! siteHasFeature( state, 'scan' ) ) {
-		return 'one-click-restores';
+	if (
+		[ 'active', 'provisioning' ].includes( rewindState ) ||
+		preflightStatus === PreflightTestStatus.SUCCESS
+	) {
+		return 'manage-security';
```

Comment:
So we want to ensure that if preflightStatus is success we will this display this sidebar instead of asking for credentials:
![CleanShot 2024-02-28 at 23 55 47@2x](https://github.com/Automattic/jetpack/assets/1488641/5de76011-957c-494a-a9de-9ef88e825b7b)